sympy/ntheory/generate.py

```diff
@@ -667,6 +667,12 @@ def primerange(a, b):
         Sieve.primerange : return range from already computed primes
                            or extend the sieve to contain the requested
                            range.
+
+        References
+        ==========
+
+        .. [1} https://en.wikipedia.org/wiki/Prime_number
```

Comment:
`.. [1]`

Reply:
Thanks! This may be the source of the weird test error. Just kept jumping between methods, while not obvious errors were found (there).